src/Stripe.ios.js

```diff
@@ -1,26 +1,31 @@
 import { NativeModules } from 'react-native'
 import processTheme from './utils/processTheme'
 import checkArgs from './utils/checkArgs'
+import checkInit from './utils/checkInit'
 import * as types from './utils/types'
 
 const { TPSStripeManager } = NativeModules
 
 class Stripe {
+  stripeInitialized = false
   init = (options = {}) => {
     checkArgs(
       types.initOptionsPropTypes,
       options, 'options', 'Stripe.init'
     )
+    this.stripeInitialized = true
     return TPSStripeManager.init(options)
   }
-  deviceSupportsApplePay = () => (
+  deviceSupportsApplePay = () => {
+    checkInit(this)
```

Comment:
You didn't add checker to `deviceSupportsAndroidPay`, but added it here. What's the difference?